src/components/NavBar/__tests__/NavBar.js

```diff
@@ -0,0 +1,8 @@
+import { NavBar } from '../NavBar';
+
+describe('Navbar component test suite', () => {
+  it('renders without crashing', () => {
+    const div1 = NavBar;
+    expect(div1).toBeDefined();
+  });
+});
```

Comment:
You should also add tests for when `<NavBar authenticated>`, renders without crashing